### ISSUE TITLE
[Bug] BlockStalagnateSeed.growsDownward apparently caused Server Crash

### ISSUE THREAD

**Issue by Xastabus** [paulevsGitch/BetterNether#502]
### What happened?

I added BetterNether to my server yesterday. At least one of my players has been bringing the new stuff back from the nether to decorate their base in the overworld (world).

Our server began crashing today when players in the overworld enter the area of the nether decorated base.

Server Log: https://mclo.gs/tbAqPuB

Crash-reports:
[crash-2022-01-19_17.36.48-server.txt](https://github.com/paulevsGitch/BetterNether/files/7899886/crash-2022-01-19_17.36.48-server.txt)
[crash-2022-01-19_17.43.00-server.txt](https://github.com/paulevsGitch/BetterNether/files/7899887/crash-2022-01-19_17.43.00-server.txt)
[crash-2022-01-19_17.46.15-server.txt](https://github.com/paulevsGitch/BetterNether/files/7899888/crash-2022-01-19_17.46.15-server.txt)

Mod List on server
[mods.txt](https://github.com/paulevsGitch/BetterNether/files/7899947/mods.txt)

Mods updates applied about 1 hour before the problem started
[updated-mods.txt](https://github.com/paulevsGitch/BetterNether/files/7899952/updated-mods.txt)

### BetterNether

6.0.8

### BCLib

1.2.4

### Fabric API

0.46.1

### Fabric Loader

0.12.12

### Minecraft

1.18.1

### Relevant log output

```shell
[17:36:48] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraft.class_148: Ticking entity
	at Not Enough Crashes deobfuscated stack trace.(1.18+build.1) ~[?:?]
	at net.minecraft.server.MinecraftServer.tickWorlds(MinecraftServer:883) ~[server-intermediary.jar:?]
	at net.minecraft.server.dedicated.MinecraftDedicatedServer.tickWorlds(MinecraftDedicatedServer:324) ~[?:?]
	at net.minecraft.server.MinecraftServer.tick(MinecraftServer:823) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.handler$zlo000$modifiedRunLoop(MinecraftServer:4856) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer:663) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer:270) ~[server-intermediary.jar:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.IllegalArgumentException: Cannot get property class_2746{name=top, clazz=class java.lang.Boolean, values=[true, false]} as it does not exist in Block{minecraft:air}
	at net.minecraft.state.State.get(State:98) ~[?:?]
	at paulevs.betternether.blocks.BlockStalagnateSeed.growsDownward(BlockStalagnateSeed.java:81) ~[better-nether-6.0.8.jar:?]
	at paulevs.betternether.blocks.BlockStalagnateSeed.canPlaceAt(BlockStalagnateSeed.java:104) ~[better-nether-6.0.8.jar:?]
	at net.minecraft.entity.ItemEntity.handler$epd00a$tick(ItemEntity:8575) ~[?:?]
	at net.minecraft.entity.ItemEntity.tick(ItemEntity:99) ~[?:?]
	at net.minecraft.server.world.ServerWorld.tickEntity(ServerWorld:699) ~[?:?]
	at net.minecraft.world.World.tickEntity(World:487) ~[?:?]
	at net.minecraft.server.world.ServerWorld.method_31420(ServerWorld:360) ~[?:?]
	at net.minecraft.world.EntityList.forEach(EntityList:54) ~[?:?]
	at net.minecraft.server.world.ServerWorld.tick(ServerWorld:328) ~[?:?]
	at net.minecraft.server.MinecraftServer.redirect$bco000$fixUpdateSuppressionCrashTick(MinecraftServer:5754) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.tickWorlds(MinecraftServer:879) ~[server-intermediary.jar:?]
	... 6 more
```

### Other Mods

```shell
Custom modpack. Provided list of mod jarfiles in previous section.
```

**Comment by quiqueck** [comment]
Thanks you for reporting. I'll look into it

**Comment by quiqueck** [comment]
A small follow up question. Is this reproducible? From the look of it the server may have had the same state as the client, as the state of the sapling block is "air" on the server side...

**Comment by Xastabus** [comment]
I rolled the server back 2 hours to before I started updates this morning.  I'll try to download the server world as it was at the time of the crashing and attempt to reproduce it locally.

**Comment by quiqueck** [comment]
The Fix should be in 6.0.10. Please re-open this issue if you still have problems after upgrading.
